core/src/main/java/com/ruslan/utils/MappingBookToDto.java

```diff
@@ -0,0 +1,28 @@
+package com.ruslan.utils;
+
+import com.ruslan.dto.BookDto;
+import com.ruslan.entity.book.Book;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MappingBookToDto {
```

Comment:
Usually this type of classes called mappers, right now it looks like method name :)
```suggestion
public class BookMapper {
```

Reply:
Also they can have static methods for this just to not add them to context, but it's unnecessary